framework/resources/Functional/amplab/amplab.json

```diff
@@ -12,7 +12,8 @@
             "schema": "dfs.drillTestDirAmplab",
```

Comment:
I know this is an example, but I don't think we want to check this in - we aren't aiming to generate output files for Amplab tests.